authority/ssh.go

```diff
@@ -101,6 +101,13 @@ func (a *Authority) GetSSHConfig(ctx context.Context, typ string, data map[strin
 		if err != nil {
 			return nil, err
 		}
+
+		// Backwards compatibility for version of the cli older than v0.18.0
+		if o.Name == "step_includes.tpl" && (data == nil || data["Version"] != "v2") {
```

Comment:
I would prepend Step to Version, but I don't want to make it StepVersion because that looks like the version number. StepTemplateVersion? Too long?